jit/v1/jit_v1.2.0-Beta-2.yaml

```diff
@@ -0,0 +1,1350 @@
+        - schema:
+            type: string
+            format: date-time
+            example: '2021-04-01T14:12:56+01:00'
+          in: query
+          name: eventCreatedDateTime
+          description: |-
+            Limit the result based on the creating date of the event. It is possible to use operators on this query parameter. This is done by adding a colon followed by an operator at the end of the queryParameterName (before the =)  eventCreatedDateTime:gte=2021-04-01T14:12:56+01:00  would result in all events created ≥ 2021-04-01T14:12:56+01:00. The following operators are supported
+            - :gte (≥ Greater than or equal)
+            - :gt (> Greater than)
+            - :lte (≤ Less than or equal)
+            - :lt (< Less than)
+            - :eq (= Equal to)
+            If no operator is provided, a strictly equal is used (this is equivalent to :eq operator).
+        - schema:
+            type: integer
+            format: int32
+            example: 100
+            default: 100
+            minimum: 1
+          in: query
+          name: limit
+          description: Maximum number of items to return.
+        - schema:
+            type: string
+            example: fE9mZnNldHw9MTAmbGltaXQ9MTA=
+          in: query
+          name: cursor
+          description: 'A server generated value to specify a specific point in a collection result, used for pagination.'
+        - schema:
+            type: string
+            example: 'carrierBookingReference:DESC'
+          in: query
+          name: sort
+          description: 'A comma-separated list of field names to define the sort order. Field names should be suffixed by a (:) followed by either the keyword ASC (for ascending order) or DESC (for descening order) to specify direction. :ASC may be omitted, in which case ascending order will be used.'
+        - schema:
+            type: string
+            example: '1'
+          in: header
+          name: API-Version
+          description: An API-Version header MAY be added to the request (optional); if added it MUST only contain MAJOR version. API-Version header MUST be aligned with the URI version.
+        - schema:
+            type: string
+            default: 'ACT,PLN,EST,REQ'
+            example: 'EST,ACT'
+          in: query
+          name: eventClassifierCode
+          description: |-
+            Code for the event classifier.
+            - PLN (Planned)
+            - ACT (Actual)
+            - REQ (Requested)
+            - EST (Estimated)
+            It is possible to select multiple values by comma (,) separating them. For multiple values the OR-operator is used. For example eventClassifierCode=EST,ACT matches both Estimated (EST) and Actual (ACT) events.Default is all eventClassifierCodes.
```

Comment:
This is a new filter - has it been added on purpose?

Reply:
eventClassifierCode is there in the filters in GET
<img width="836" alt="image" src="https://github.com/dcsaorg/DCSA-OpenAPI/assets/138600841/baa91b5c-5f34-4769-a4a7-2660e1133b90">